packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -144,6 +146,49 @@ def test_if_we_have_capabilities(cluster):
     assert {'name': 'PACKAGE_MANAGEMENT'} in r.json()['capabilities']
 
 
+def test_cosmos_package_add(cluster):
+    def package_add_configured():
+        user_config = load_json("/opt/mesosphere/etc/expanded.config.json")
```

Comment:
Please only use inner functions if they must take advantage of local variables. There are no such variables being used here